misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/SuspendedContext.java

```diff
@@ -55,16 +59,14 @@
     private ClassLoaderReference classLoader;
     private DebugExpressionCompiler debugCompiler;
 
-    SuspendedContext(Project project, VirtualMachineProxyImpl vm, ThreadReferenceProxyImpl threadRef,
+    SuspendedContext(ExecutionContext executionContext, ThreadReferenceProxyImpl threadRef,
                      StackFrameProxyImpl frame) {
-        this.attachedVm = vm;
+        this.executionContext = executionContext;
```

Comment:
Seems like there is no usage for `executionContext ` here. Shall we remove it?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/30249/commits/b757b9606646c2a17a343cbd0fc80347236265a6